src/core/InformationPanel/InformationPanel.tsx

```diff
@@ -0,0 +1,174 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { IconButton } from '../Buttons';
+import { CommonProps, useMergedRefs, useTheme } from '../utils';
+import SvgCloseSmall from '@itwin/itwinui-icons-react/cjs/icons/CloseSmall';
+import '@itwin/itwinui-css/css/information-panel.css';
+
+export type InformationPanelProps = {
+  /**
+   * Label shown at the top of the panel.
+   */
+  label?: React.ReactNode;
+  /**
+   * Is the panel open?
+   * @default false
+   */
+  isOpen?: boolean;
+  /**
+   * Orientation of the panel.
+   * @default 'vertical'
+   */
+  orientation?: 'vertical' | 'horizontal';
+  /**
+   * Callback fired when close icon is clicked.
+   */
+  onClose?: () => void;
+  /**
+   * Custom header action buttons shown alongside the close button.
+   * @example
+   * [
+   *   <IconButton styleType='borderless' onClick={() => {}}><SvgEdit /></IconButton>,
+   *   <IconButton styleType='borderless' onClick={() => {}}><SvgDelete /></IconButton>,
+   * ]
+   */
+  headerActions?: React.ReactNode[];
+  /**
+   * Set to false to make the panel non-resizable.
```

Comment:
But default is also falso, so maybe you meant `set to true to make the panel resizable` ?